features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/MessagesNode.kt

```diff
@@ -135,16 +135,20 @@ class MessagesNode @AssistedInject constructor(
                 callbacks.forEach { it.onUserDataClick(permalink.userId) }
             }
             is PermalinkData.RoomLink -> {
-                handleRoomLinkClick(permalink, eventSink)
+                handleRoomLinkClick(activity, permalink, eventSink)
             }
             is PermalinkData.FallbackLink,
             is PermalinkData.RoomEmailInviteLink -> {
-                context.openUrlInExternalApp(url)
+                activity.openUrlInChromeCustomTab(null, darkTheme, url)
```

Comment:
Note: at some point, direct usage of `openUrlInExternalApp` should be forbidden, but this is out of scope of this PR.